.github/actions/build-gauntlet/action.yml

```
@@ -0,0 +1,29 @@
+name: Install gauntlet dependencies
+description: A GitHub Action to get tool versions and build Gauntlet
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4.1.5
+
+    - name: Get Tool Versions
+      uses: smartcontractkit/tool-versions-to-env-action@v1.0.8
+      id: tool-versions
+
+    - name: Setup Node ${{ steps.tool-versions.outputs.nodejs_version }}
+      uses: actions/setup-node@v4.0.2
+      with:
+        node-version: ${{ steps.tool-versions.outputs.nodejs_version }}
+
+    - name: Install Dependencies
+      run: yarn --cwd ./gauntlet install --frozen-lockfile
+      shell: bash
```

Let's please start standardising on Nix for shell environments:

```suggestion
    - name: Install Dependencies
      run: nix develop -c yarn -- --cwd ./gauntlet install --frozen-lockfile
      shell: bash
```

---

@smickovskid can you make a follow up ticket for this?

---

Created